src/applications/search/containers/SearchApp.jsx

```diff
@@ -96,27 +103,19 @@ class SearchApp extends React.Component {
     });
 
     // Fetch new results
-    this.props.fetchSearchResults(userInput, nextPage);
+    this.props.fetchSearchResults(userInput, nextPage, {
+      path: document.location.pathname,
+      userInput,
+      typeaheadEnabled: undefined,
```

Comment:
maybe a comment here explaining why we're setting all these explicitly to `undefined`?